lib/form/scripts.js

```diff
@@ -24,6 +24,17 @@ const rendered = (type) => {
     let plugins = type.attributes.map(plugin => {
       return plugin.type;
     });
+
+    // Checks if plugin directory has values
+    if (!Array.isArray(dir)) {
+      rej(new Error('Plugin directory congiguration must be an array'));
+    }
+
+    // add node_modules if it's not in there
+    if (dir.indexOf(path.join(__dirname, '../../node_modules')) === -1) {
```

Comment:
Something changed in the latest npm install. Happened to ayush too

On Aug 1, 2016 2:32 PM, "Sam Richard" <notifications@github.com> wrote:

> In lib/form/scripts.js
> <https://github.com/punchcard-cms/content-types/pull/72#discussion_r73028687>
> :
>
> > @@ -24,6 +24,17 @@ const rendered = (type) => {
> >      let plugins = type.attributes.map(plugin => {
> >        return plugin.type;
> >      });
> > +
> > +    // Checks if plugin directory has values
> > +    if (!Array.isArray(dir)) {
> > +      rej(new Error('Plugin directory congiguration must be an array'));
> > +    }
> > +
> > +    // add node_modules if it's not in there
> > +    if (dir.indexOf(path.join(__dirname, '../../node_modules')) === -1) {
>
> Where did this stuff come from? This is totally out-of-scope for
> identifier *and* it's grabbing the wrong node_modules folder
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/punchcard-cms/content-types/pull/72/files/e558bb4874f598a0ce6e98dae5225cc6effbc962#r73028687>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AANPY8xq7AV1FxtEpNrztbEjdIQK6jAjks5qbjvEgaJpZM4JVohz>
> .
>